### PR TITLE
Added the new function

### PR DESCRIPTION
**Pull request by Nathan-E**
Please review the current function added

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -25,3 +25,18 @@ console.log(AddNumbers([2,1,-3,-4]));
...
+// Addition and Multiplication in one Function
+function AddandMultiply (arr) {
+	let add = 0;
+	let multi = 1;
+
+	for (let i = 0; i < arr.length; i++) {
+		add += arr[i];
+		multi *= arr[i];		
+	}
+	return `Addition: ${add} \nMultiply: ${multi}`;
```
Comment (nnamdi16) [inline index.js]: Create space between \n and Multiply for improved readability

**Review by nnamdi16 — COMMENTED** [review]
Go through the earlier comment I made

File: index.js (hunk 1 above)
Comment (Nathan-E) [reply index.js]: ok

**Comment by Nathan-E** [comment]
review for changes made

**Review by nnamdi16 — APPROVED** [review]
Approved

**Pull request merged**